libcxx/include/module.modulemap

```diff
@@ -2285,6 +2285,7 @@ module std [system] {
   }
   module internal_assert {
     header "__assert"
+    header "__assertion_handler" // Generated via CMake
```

Comment:
This should be in a separate submodule - in general each submodule should have only a single `header` statement, otherwise you "fuse" the two headers. i.e. wherever you have `#include <__assertion_handler>` it'll also include `__assert` which is at least unintuitive.